Clappr/Classes/Base/Player.swift

```diff
@@ -127,10 +127,10 @@ open class Player: BaseObject {
         core?.activePlayback?.seek(timeInterval)
     }
 
-    open func setFullscreen(_ fullscreen: Bool) {
```

Comment:
I recommend keeping the `setFullscreen` external interface as it is used by all other apps.